src/assets/ocisWrapper/ocisWrapper.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Automated Black-Box Tests for System Settings of a Microservice Application
+authorName: Artur Neumann
+authorAvatar: https://www.jankaritech.com/images/2019/06/11/p1070364-c-light-800.jpg
+authorLink: https://www.linkedin.com/in/artur-neumann/
+createdAt: May 22, 2023
+tags: go, API, testing, microservice
+banner: https://github.com/JankariTech/blog/blob/master/src/assets/ocisWrapper/banner.png
+---
+oCIS is the new generation open source file-sync and sharing platform, built by [ownCloud GmbH](https://www.owncloud.com), who also created the ownCloud 10 server based on the LAMP stack. The team of [JankariTech](https://www.jankaritech.com) is part of that development and mainly writes and maintains the automated tests.
+
+## The Challenge 🚀
+
+Because a lot of APIs of oCIS still have to be compatible with ownCloud 10, we have used the existing API tests and (after some adjustment) ran them on oCIS to ensure the backwards compatibility. I have written about that process in [BDD on Software Rewrite](https://blog.jankaritech.com/#/blog/Behaviour%20Driven%20Development/BDD%20on%20Software%20Rewrite).
```

Comment:
```suggestion
## The Challenge 🚀

Because a lot of APIs of oCIS still have to be compatible with ownCloud 10, we have used the existing API tests and (after some adjustments) ran them on oCIS to ensure backward compatibility. I have written about that process in [BDD on Software Rewrite](https://blog.jankaritech.com/#/blog/Behaviour%20Driven%20Development/BDD%20on%20Software%20Rewrite).
```